tests/build_rpms.sh

```diff
@@ -20,10 +20,13 @@ rpm -U https://dl.fedoraproject.org/pub/epel/epel-release-latest-${OS_VERSION}.n
 # Broken mirror?
 echo "exclude=mirror.beyondhosting.net" >> /etc/yum/pluginconf.d/fastestmirror.conf
 
-yum -y -d0 install yum-plugin-priorities rpm-build gcc gcc-c++ boost-devel cmake git tar gzip make autotools openssl
+yum -y -d0 install rpm-build gcc gcc-c++ boost-devel cmake git tar gzip make autotools openssl python3 python-srpm-macros python-rpm-macros python3-rpm-macros python3-devel rrdtool rrdtool-devel
```

Comment:
Why do we need these extra packages for the RPM build

Reply:
The various python*-macros packages define the `%{__python}` and related macros that are used in the spec file.

Reply:
Are the non-python3 versions required for that, too?

Reply:
Actually I think _only_ the non-python3 macros are required -- for example, we use `%{python_sitelib}` instead of `%{python3_sitelib}`. Mark, can you try it without python3-rpm-macros?